client/app/lib/remote-extensions/machine.coffee

```diff
@@ -286,9 +278,9 @@ module.exports = class JMachine extends remote.api.JMachine
     { Own, Shared, Reassigned, Collaboration } = JMachine.Type
 
     switch
-      when @isMine()      then Own
       when @isPermanent() then Shared
       when @getOldOwner() then Reassigned
+      when @isMine()      then Own
```

Comment:
does `isMine` not working correctly for disabled stacks? if so, it might be better to handle/fix it in there.